common/src/main/java/io/netty/util/internal/TypeParameterMatcher.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013 The Netty Project
+ * Copyright 2021 The Netty Project
```

Comment:
It should be 2013 as this class was created back in 2013. WDYT @normanmaurer?

Reply:
Yea please revert @terrarier2111

Reply:
I reverted the change.